.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish a release
+
+on:
+  push:
```

Comment:
How about [release](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#release) ?

It's not strictly the same thing, it's triggered by publishing (directly or from a release draft) from https://github.com/ekino/veggies/releases/new, but it forces us to correctly draft a release note when tagging (tag can be created through the release process).

Reply:
After discussion all the action stand in `release.yml` 
Bump - Changleog - Commit - Push - Tag - Release - Publish